src/stylesheets/elements/_inputs.scss

```diff
@@ -77,8 +77,19 @@ select {
   label {
     margin-top: 0;
   }
+
+  .usa-input-inline {
+    border: $input-border-width solid $color-gray;
+    width: inherit;
+  }
+
+  .usa-input-inline-error {
+    border: 3px solid $color-secondary-dark;
```

Comment:
Let's pair on this to see where the problem is happening, bc it looks like the image here https://github.com/18F/web-design-standards/pull/1251#issuecomment-225316504 with the above code on my machine.